src/cli/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Prefix your items with `(Template)` if the change is about the template and not
 
 ## 0.19.0
 - Replaced Powershell Copy Tool by a Dart CLI.
+- Target specific Dart/Flutter SDK.
```

Comment:
This should have an impact on the version. You're not changing the CI of the template, but rather the CI of the generated application. You'll want a new version.

Reply:
I'm doing both but you are right!